sqlmesh/core/scheduler.py

```diff
@@ -193,6 +193,7 @@ def run(
         restatements: t.Optional[t.Dict[str, SnapshotInterval]] = None,
         ignore_cron: bool = False,
         selected_snapshots: t.Optional[t.Set[str]] = None,
+        circuit_breaker: t.Optional[t.Callable[[], bool]] = None,
```

Comment:
Is this used?

Reply:
yes, we now pass `_has_environment_changed` as a circuit breaker